### ISSUE TITLE
Access to user manuals via the app

### ISSUE THREAD

**Issue by ColinSloane** [AppertaFoundation/Diadem#40]
we need to bring all the good content over to UserVoice (an incorporate that nicely into the
app too) - move to new requirement this is not linked to title above

**Comment by ApplicationInsight** [comment]
This issue replicates issue #24

**Comment by ColinSloane** [comment]
I have updated the quick guide flowchart to replicate the 1 to 5 steps we have agreed the App version will follow which replicates the paper version.  See attached please let me know if this looks OK

Dave I am suggesting we follow 1 Functional Impairment with 2 Cognitive Impairment test (6 CIT or GP COG) and then present the Informant screen.

![quick guide flowchart v2](https://user-images.githubusercontent.com/30338798/29072666-81403ae0-7c40-11e7-85be-ce1021d8b974.jpg)

Let me know if this seems OK and I will get the user guides finished off

**Comment by ApplicationInsight** [comment]
Colin yes flowchart looks much better [smile]

If we were to continue with the same logic we currently use, The new running order of screens would be something like - if 6 CIT is chosen the informant screen will follow directly after the COG test. If GP COG is chosen the informant screen will follow part 1 before going on to part 2 (if the patient's score warrants this), or will follow on directly in the same fashion as 6 CIT

In other words, the informant screen would always appear after the 5 elements of DiADeM have begun - might it be better to have the informant screen appear between 'Location' and 'Functional Impairment'?

**Comment by ColinSloane** [comment]
Thanks for that Dave - I would suggest we keep it where I have put it in the revised diagram - just because if you get to the end of either Cog test and its FALSE you don't need the Informant for any background info I.E the corroborating History stuff, if that makes sense?

I see what your saying about the difference re GP Cog Informant interview needed but either way the next step after the cog test is either 'end test' or 'The Informant' so 

- 6CIT = True goto                                          The Informant
- GP COG pt1 = True goto                              The Informant
- GP COG pt1 =  need pt2 = True goto          The Informant
- Any of the above = False goto                     Review

**Comment by ApplicationInsight** [comment]
Sure - and yes for the user there would be no effective difference (unless part 2 of GP COG is needed) - it was more whether you wanted to have the 'admin stuff' concerning the informant to appear during the DiADeM elements or to cover it before along with other 'admin stuff'...

Easier for me to keep it where it is, just thought I'd ask...

**Comment by ApplicationInsight** [comment]
OK fine [smile]

**Comment by ColinSloane** [comment]
Great Stuff Dave [Smile - Smile]

**Comment by ColinSloane** [comment]
Hi ALL 
The user guides are now complete ad have been updated to reflect the NEW flow withing the App which is now in line with the paper version and goes through the tests 1 to 5
Have uploaded word doc and PDF format for each one

[DiADeM Final quick guide 'Clinicians proposing to undertake DiADeM app assessments.docx](https://github.com/AppertaFoundation/Diadem/files/1214777/DiADeM.Final.quick.guide.Clinicians.proposing.to.undertake.DiADeM.app.assessments.docx)
[DiADeM Final quick guide 'Clinicians proposing to undertake DiADeM app assessments.pdf](https://github.com/AppertaFoundation/Diadem/files/1214775/DiADeM.Final.quick.guide.Clinicians.proposing.to.undertake.DiADeM.app.assessments.pdf)
[DiADeM Final quick guide report recipients.docx](https://github.com/AppertaFoundation/Diadem/files/1214774/DiADeM.Final.quick.guide.report.recipients.docx)
[DiADeM Final quick guide report recipients.pdf](https://github.com/AppertaFoundation/Diadem/files/1214776/DiADeM.Final.quick.guide.report.recipients.pdf)
[DiADeM testing part 2 Manual draftv3.docx](https://github.com/AppertaFoundation/Diadem/files/1214778/DiADeM.testing.part.2.Manual.draftv3.docx)
[DiADeM testing part 2 Manual draftv3.pdf](https://github.com/AppertaFoundation/Diadem/files/1214779/DiADeM.testing.part.2.Manual.draftv3.pdf)

**Comment by ColinSloane** [comment]
Colin to write in the NEW save and Exit bits to the manuals and then upload again for Dave to add to the Info button within the App deadline Thursday 31st
